gammapy/spectrum/spectrum_results.py

```diff
@@ -203,6 +203,20 @@ def from_json(data):
 
         return model
 
+    @staticmethod
+    def from_sherpa(covar):
+        """Read sherpa fit output
+
+        Parameters
+        ----------
+        covar : `~sherpa.fit.ErrorEstResults`
+            Fit results as return by :func:`~sherpa.astro.datastack.covar`
+        """
+        model = SpectralModelPowerLaw()
+        model.meta = Bunch(covar.__dict__)
```

Comment:
I also thought about it, my suggestion would be
* Leave the meta attribute as it is in order to save all available information
* Introduce attributes for the most important parameters and gradually add the ones I need. For now this would probably be
```
model.gamma
model.gamma_err
model.norm
model.norm_err
```
These would need to be filled in the from_json and from_sherpa methods. I guess that would be a start.

What do  you think?